libpod/container_copy_linux.go

```diff
@@ -48,7 +48,11 @@ func (c *Container) copyFromArchive(path string, chown bool, rename map[string]s
 		if err != nil {
 			return nil, err
 		}
-		unmount = func() { c.unmount(false) }
+		unmount = func() {
+			if err := c.unmount(false); err != nil {
+				logrus.Errorf("failed to unmount container: %v", err)
```

Comment:
The probably inconsistent convention is to have logs capitalized to make them easier to brain parse:

```suggestion
				logrus.Errorf("Failed to unmount container: %v", err)
```

Reply:
Same for all other places.

Reply:
I guess it is time to find a linter for that as well :)